### PR TITLE
Change: page add employee

### PR DESCRIPTION
**Pull request by LeonidErshov**

### CODE REVIEW

File: components/EmployeeComponents/AddEmployee.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import { Button, Form, Input, message } from "antd";
+import { Button, Form, Input, message, Typography, Checkbox } from "antd";
```
Comment (gaskeo) [inline components/EmployeeComponents/AddEmployee.tsx:2]: delete unused `Typography` and `Checkbox`

File: components/EmployeeComponents/AddEmployee.tsx [hunk 2]
```diff
@@ -38,54 +38,75 @@ export default function AddEmployee() {
...
         <>
-            <div className={styles.title}>Добавление сотрудника</div>
-            <Form
-                onFinish={onFinish}
-                onFinishFailed={onFinishFailed}
-                className={styles.container}
-            >
+            <div className={styles.container}>
+            <h1 className={styles.title}> Добавление сотрудника </h1>
+            <Form onFinish={onFinish}>
```
Comment (gaskeo) [inline components/EmployeeComponents/AddEmployee.tsx:43]: `onFinishFailed` (row 36) unused. It required in this `Form` component. Bring back this function in prop or delete it.

File: components/EmployeeComponents/AddEmployee.tsx [hunk 3]
```diff
@@ -38,54 +38,75 @@ export default function AddEmployee() {
...
-                    label="Должность"
-                    name="position"
+                    label="Эл.почта"
+                    name="email"
+                    className={styles.label}
                     rules={[
                         {
-                            required: true,
-                            message: "Пожалуйста введите должность",
+                            required: false,
```
Comment (gaskeo) [inline components/EmployeeComponents/AddEmployee.tsx:96]: Is email really not required?

File: components/EmployeeComponents/styles/AddEmployee.module.scss [hunk 4]
```diff
@@ -1,10 +1,38 @@
...
+.label label {
+    width: 150px;
+    font-size: 16px;
+}
+
+.input {
+    max-width: 372px;
+}
+
+/*.formBtn{
```
Comment (gaskeo) [inline components/EmployeeComponents/styles/AddEmployee.module.scss:26]: If it isn't used - delete it.

**Review by gaskeo — CHANGES_REQUESTED** [review]

File: components/EmployeeComponents/AddEmployee.tsx (hunk 3 above)
Comment (LeonidErshov) [reply components/EmployeeComponents/AddEmployee.tsx:96]: Yes. In Figma so

**Pull request closed without merging**